README.md

```diff
@@ -1,2 +1,49 @@
-# Book-Store
-The Bookstore is a website similar to the "Awesome Books" website built in the previous module. I will create an MVP version of it that allows the user to:  Display a list of books. Add a book. Remove a selected book. 
+# Getting Started with Create React App
+
+Book-Store
+The Bookstore is a website similar to the "Awesome Books" website built in the previous module. I will create an MVP version of it that allows the user to: Display a list of books. Add a book. Remove a selected book.
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+## Mini Demo
+not available yet
+
+## Screenshot
+not available yet
+
+Additional description about the project and its features.
+
+## Built With
+Major languages: Html Css JS Frameworks: React Technologies used : Visual Studio Code
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+clone the repo using this command : git clone git@github.com:brahimdidi/math-magic.git opent the project with your favourite code editor(vscode, athom...) run npm install run npm start
+
+## Prerequisites
+A text editor(preferably Visual Studio Code) Install Git Node Usage Clone this repository git clone git@github.com:brahimdidi/math-magic.git cd math-magic Run project npm start Open page in browser $ open dist/index.html Author bust_in_silhouette Brahim
+
+## GitHub:
+ @brahimdidi Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support Give a star if you like this project!
+
+memo License This project is MIT licensed.
```

Comment:
- [ ] Kindly update your readme file following the [Readme Template](https://github.com/microverseinc/readme-template/blob/master/README.md) and add all main headers from the template.

Reply:
done